feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/SessionDetailFragment.kt

```diff
@@ -99,6 +100,10 @@ class SessionDetailFragment : DaggerFragment() {
             container,
             false
         )
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            activity?.window?.decorView?.systemUiVisibility =
```

Comment:
We are using SystemUiManager class. can you check it? 🙏

Reply:
@takahirom 
sorry. I was busy working and couldn't help.
Unfortunately I couldn't call SystemUiManager.
Could you give me any advice?

Reply:
Of course! 
How about moving SystemUiManager to android corecomponet?

Reply:
@takahirom 
This is solved using SystemUiManager! Thank you for the advice!
 commit a08be2aaf8c3692050dcfd3c88c0ff2ecdbe4c3d